tests/PHPStan/Rules/Arrays/NonexistentOffsetInArrayDimFetchRuleTest.php

```diff
@@ -700,4 +700,15 @@ public function testBug6243(): void
 		$this->analyse([__DIR__ . '/data/bug-6243.php'], []);
 	}
 
+	public function testBug8356(): void
+	{
+		$this->bleedingEdge = true;
+		$this->analyse([__DIR__ . '/data/bug-8356.php'], [
+			[
+				"Offset 'x' might not exist on array|string.",
```

Comment:
before this PR, the rule reported 
`Offset 'x' does not exist on array|string.`
.. now we get  ..
`Offset 'x' might not exist on array|string.`

when running this test without `$this->bleedingEdge = true;` the signature changes in the stub do not have any effect and we still get `Offset 'x' does not exist on array|string.`

Reply:
The bot only goes through triaged issues labeled as bug or feature-request.